### PR TITLE
switch to sass/dvl

### LINKED ISSUE #158: Switch to SASS
cf. #157

### PR DESCRIPTION
**Pull request by aviv**
closes #158

@kariannemah if you want to pull this down and see if all the styles look OK or if I missed anything, that’d be awesome.

### CODE REVIEW

**Comment by kariannemah** [comment]
Since I'm still learning how CSS is done post-2001, I pulled this down to see how the styles are rendered in the views. Nothing stuck out as odd.

:+1: from me

**Comment by ajb** [comment]
dvl-core needs to be included as a dependency -- you should not be including the files directly.

**Comment by aviv** [comment]
Ty both, I'll add the dep and merge tonight.

**Comment by ajb** [comment]
Ping me to review/merge, plz.

On Tue, Feb 24, 2015 at 6:57 PM, Aviv Nitsan <notifications@github.com>
wrote:

> Ty both, I'll add the dep and merge tonight.
>
> On Tue, Feb 24, 2015, at 03:36 PM, Adam Becker wrote:
> > dvl-core needs to be included as a dependency -- you should not be
> > including the files directly.
> >
> > ---
> > Reply to this email directly or view it on GitHub:
> > https://github.com/dobtco/knowledge_base/pull/166#issuecomment-75874033
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/knowledge_base/pull/166#issuecomment-75877731>.
>



-- 
Adam Becker
(951) 9-BECKER
@AdamJacobBecker

File: contents/styles/core.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@import "core/includes";
```
Comment (jrubenoff) [inline contents/styles/core.scss]: Why not `@import core/**`?

File: contents/styles/knowledge_base.scss [hunk 2]
```diff
@@ -0,0 +1,651 @@
...
+  min-width: 30rem;
+  padding: 1rem;
+}
+
+code {
+  display: inline-block;
+  border: 0;
+  background: #eee;
+  color: #32713E;
+  backround: transparent;
```
Comment (jrubenoff) [inline contents/styles/knowledge_base.scss]: Watch it

Comment (aviv) [inline contents/styles/knowledge_base.scss]: Thanks!

**Comment by aviv** [comment]
I'm actually not quite sure the best way to include `dvl-core` as a dependency. Tried [this approach](https://docs.npmjs.com/files/package.json#github-urls) but didn't seem to be right. Maybe we need to set up `dvl-core` (or a branch?) so that it can be included properly as a dependency in a node/wintersmith environment?

**Comment by ajb** [comment]
Correct, dvl-core is a Ruby gem, not an npm package. 

What about just using one of the compiled versions in https://github.com/dobtco/dvl-core/tree/master/dist?

**Comment by ajb** [comment]
Did some minor hacking to get wintersmith to add the bundle'd `dvl-core` to the load path. Seems to work fine on both my machine and CircleCI. If we were to start from the beginning, I'd probably have us do this in Ruby instead of node for easier compatibility with the dvl-core and rest of our stack, but still, it's nice to have the ability to use multiple languages.

Anyway... this seems ready to me.

**Comment by aviv** [comment]
Just a bit weird that we're using bundler/gems in a node environment, but seems to work fine so :+1:

**Comment by ajb** [comment]
Yes and no -- the only thing that matters is that it's easy to install the dependencies on our development machines :white_check_mark:, and that CircleCI which is building and deploying the site handles both. :white_check_mark:

**Pull request merged**